queries/matlab/indents.scm

```diff
@@ -0,0 +1,32 @@
+((keyword) @indent.end @indent.branch
+  (#eq? @indent.end "end")
+  (#eq? @indent.branch "end"))
+
+[(if_statement)
+ (for_statement)
+ (while_statement)
+ (switch_statement)
+ (try_statement)
+ (function_definition)
+ (class_definition)
+ (enumeration)
+ (events)
+ (methods)
+ (properties)
+ ] @indent.begin
+
+[(elseif_statement)
+ (else_statement)
+ (case)
+ (otherwise)
+ (catch)
+ ] @indent.branch
+
+((matrix_definition (row) @indent.align)
+ (#set! indent.open_delimiter "["))
+((cell_definition (row) @indent.align)
+ (#set! indent.open_delimiter "{"))
+((parenthesized_expression) @indent.align
+ (#set! indent.open_delimiter "("))
```

Comment:
Have you done some tests locally and found out that not having closing delimiter results in more correct indentation? If yes, then I'm okay with this, if not, please add their respective closing braces.

Reply:
I didn't know about the closing delimiters, I followed the code I found in this repo and it worked, so I left it like that. I'll add the `close_delimiter` now.

Reply:
I added it. Made no difference when testing, though.